src/pages/Details/Detail.scss

```
@@ -0,0 +1,202 @@
+.detail {
+  width: 1351px;
+  margin: 0 auto;
+  background-color: #ffffff;
+  .contents {
```

```suggestion
  background-color: #ffffff;
  
  .contents {
```

각 선택자 부분은 가독성을 위해서 한칸씩 띄워주세요!